### ISSUE TITLE
invMass to energyBranch Name code duplication

### ISSUE THREAD

**Issue by ldcorpe** [ECALELFS/ECALELF#45]
This is not urgent, but there is a duplication of nearly identical code which should be factorised, since otherwise if one wants to add a new energy Branch, one needs to change it ins several places instead of just one. 

The two places are:
https://github.com/ECALELFS/ECALELF/blob/3c7c7fc5f2b6976912312712d049fff3823e8896/ZFitter/src/ZFit_class.cc#L1175
and:
https://github.com/ECALELFS/ECALELF/blob/3c7c7fc5f2b6976912312712d049fff3823e8896/ZFitter/bin/ZFitter.cpp#L212

**Comment by shervin86** [comment]
I agree, maybe we should create a library of functions that can be loaded by both.
